vsphere-mixin/panels.libsonnet

```diff
@@ -0,0 +1,1472 @@
+            table.standardOptions.withUnit('rotmhz')
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('CPU utilization')
+          + fieldOverride.byName.withProperty('custom.displayMode', 'gradient-gauge')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('custom.width', 157)
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('percent')
+            + table.standardOptions.color.withMode('continuous-BlPu')
+            + table.standardOptions.withMin(0)
+            + table.standardOptions.withMax(100)
+            + table.standardOptions.withDecimals(1)
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Memory usage')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('custom.width', 140)
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('mbytes')
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Memory utilization')
+          + fieldOverride.byName.withProperty('custom.displayMode', 'gradient-gauge')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('custom.width', 157)
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('percent')
+            + table.standardOptions.color.withMode('continuous-BlPu')
+            + table.standardOptions.withMin(0)
+            + table.standardOptions.withMax(100)
+            + table.standardOptions.withDecimals(1)
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Disk usage')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('custom.width', 140)
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('bytes')
+          ),
+        ])
+        + table.standardOptions.withOverridesMixin([
+          fieldOverride.byName.new('Disk utilization')
+          + fieldOverride.byName.withProperty('custom.displayMode', 'gradient-gauge')
+          + fieldOverride.byName.withProperty('custom.align', 'left')
+          + table.fieldOverride.byName.withProperty('custom.width', 157)
+          + fieldOverride.byName.withPropertiesFromOptions(
+            table.standardOptions.withUnit('percent')
+            + table.standardOptions.color.withMode('continuous-BlPu')
+            + table.standardOptions.withMin(0)
+            + table.standardOptions.withMax(100)
+            + table.standardOptions.withDecimals(1)
+          ),
```

Comment:
The gradient for the disk utilization panel feels a little overly aggressive, and personally I'm not sure about the colour scheme.

I'd expect something more akin to green -> yellow -> red, rather than blue with interrupted yellows, but I think it's just a case of playing about a little

![image](https://github.com/grafana/jsonnet-libs/assets/1282515/6e796343-d73e-4092-9abf-edd2b83ae5a9)

Reply:
@Dasomeone the green to red actually seems more aggressive iMO haha? 
<img width="2121" alt="image" src="https://github.com/grafana/jsonnet-libs/assets/12396806/a7db66f8-c330-4cec-83f1-783040a180e4">
I think we've used blue to red on the last few integrations. I believe it's actually mentioned in the style guide as well for gradients.

Reply:
In that case please ignore my message! Clearly I need to double check our own docs :sweat_smile: